src/backend/SecurityAction.h

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2025 The Johns Hopkins University Applied Physics
+ * Laboratory LLC.
+ *
+ * This file is part of the Bundle Protocol Security Library (BSL).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * This work was performed for the Jet Propulsion Laboratory, California
+ * Institute of Technology, sponsored by the United States Government under
+ * the prime contract 80NM0018D0004 between the Caltech and NASA under
+ * subcontract 1700763.
+ */
+#include <m-array.h>
+#include <BPSecLib_Private.h>
+#include "SecOperation.h"
+
+// NOLINTBEGIN
+ARRAY_DEF(BSL_SecOperList, BSL_SecOper_t, M_OPEXTEND(M_POD_OPLIST, CLEAR(API_2(BSL_SecOper_Deinit))))
```

Comment:
I don't think this can legitimately inherit `INIT_SET`, `SET` and others from POD because it is itself not plain data.